sudoku/hard/addRandomHard.js

```diff
@@ -0,0 +1,28 @@
+export function addRandom(div) {
```

Comment:
what's the difference between the hard addRandom and the easy one?

Reply:
no difference, it just doesn't work correctly when  imported from separate files.